kubemarine/procedures/restore.py

```diff
@@ -201,7 +201,8 @@ def import_etcd(cluster: KubernetesCluster) -> None:
         control_plane_conn.sudo(
             f'chmod 777 {snap_name} && '
             f'sudo ls -la {snap_name} && '
-            f'sudo etcdctl snapshot restore {snap_name} '
+            f'sudo ETCD_IMAGE="{etcd_image}" ETCD_CERT="{etcd_cert}" ETCD_KEY="{etcd_key}" ETCD_CA="{etcd_cacert}" '
+            f'ETCD_ENDPOINTS="{initial_cluster}" ETCD_MOUNTS="{mount_options}" etcdctl snapshot restore {snap_name} '
```

Comment:
ETCD_ENDPOINTS is passed to `--endpoints`. `initial_cluster` variable is passed for `--initial-cluster` and has a value of different format than for `--endpoints`

Also, there already is an ability to override default ETCD_ENDPOINTS variable by `etcdctl --endpoints`. If we want to hardcode this flag, can we use it? https://github.com/Netcracker/KubeMarine/blob/main/kubemarine/resources/scripts/etcdctl.sh#L58

Reply:
Fixed in https://github.com/Netcracker/KubeMarine/pull/651/commits/38b8bc048059c7fefb6cae80fc0bc3f949a97742